--- /home/user/prismhookv2/contracts/target/lib/solady/src/utils/LibBytes.sol	2026-08-16 03:12:10.661596737 +0000
+++ gh/LibBytes.sol.main	2026-08-16 03:25:28.118820176 +0000
@@ -170,7 +170,9 @@
             if iszero(gt(needleLen, mload(subject))) {
                 let subjectSearchEnd := add(sub(mload(0x00), needleLen), 1)
                 let h := 0 // The hash of `needle`.
-                if iszero(lt(needleLen, 0x20)) { h := keccak256(add(needle, 0x20), needleLen) }
+                if iszero(lt(needleLen, 0x20)) {
+                    h := keccak256(add(needle, 0x20), needleLen)
+                }
                 let s := mload(add(needle, 0x20))
                 for { let m := shl(3, sub(0x20, and(needleLen, 0x1f))) } 1 {} {
                     let t := mload(i)
@@ -514,11 +516,7 @@
 
     /// @dev Reduces the size of `subject` to `n`.
     /// If `n` is greater than the size of `subject`, this will be a no-op.
-    function truncate(bytes memory subject, uint256 n)
-        internal
-        pure
-        returns (bytes memory result)
-    {
+    function truncate(bytes memory subject, uint256 n) internal pure returns (bytes memory result) {
         /// @solidity memory-safe-assembly
         assembly {
             result := subject
@@ -556,7 +554,9 @@
                 let o := add(result, 0x20)
                 let subjectSearchEnd := add(sub(add(i, mload(subject)), searchLen), 1)
                 let h := 0 // The hash of `needle`.
-                if iszero(lt(searchLen, 0x20)) { h := keccak256(add(needle, 0x20), searchLen) }
+                if iszero(lt(searchLen, 0x20)) {
+                    h := keccak256(add(needle, 0x20), searchLen)
+                }
                 let s := mload(add(needle, 0x20))
                 for { let m := shl(3, sub(0x20, and(searchLen, 0x1f))) } 1 {} {
                     let t := mload(i)
@@ -782,11 +782,7 @@
     }
 
     /// @dev Returns the word at `offset`, without any bounds checks.
-    function loadCalldata(bytes calldata a, uint256 offset)
-        internal
-        pure
-        returns (bytes32 result)
-    {
+    function loadCalldata(bytes calldata a, uint256 offset) internal pure returns (bytes32 result) {
         /// @solidity memory-safe-assembly
         assembly {
             result := calldataload(add(a.offset, offset))
